client/src/components/styles/CommonStyle.scss

```diff
@@ -31,17 +37,15 @@
     height: 1px;
     background: $color-grey;
     width: 34%;
-    margin: 5px;
-    margin-left: 10px;
-    margin-right: 10px;
+    margin: 5px 10px 0 10px;
   }
 }
 
 .main-phone {
   position: absolute;
   background: center / contain no-repeat url(../assets/main-phone.png);
-  height: 70%;
+  height: 600px;
```

Comment:
what about mobile? Does it look good with hardcoded px?

Reply:
On small screen, i believe, there will be no cat at all. We hardcoded it, because if we put height in percentage, it would be changing constantly, because other column, on which it's height depends, changes it's height, when there is validation errors or loading spinner.